### PR TITLE
updates content width on filter list in modal window

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2015-12-01 at 08 51 14|screen shot 2015-12-01 at 08 51 23|

### CODE REVIEW

File: djangocms_admin_style/sass/components/_modal.scss [hunk 1]
```diff
@@ -57,7 +57,7 @@
                 // make sure that changelist form is floated left next to filter #238
                 #changelist-form {
                     float: left;
-                    width: calc(100% - 220px) !important;
+                    width: calc(100% - 248px) !important;
```
Comment (vxsx) [inline djangocms_admin_style/sass/components/_modal.scss]: is it possible to extract `248px` into a variable?

**Comment by FinalAngel** [comment]
didn't we have this issue once already?

File: djangocms_admin_style/sass/components/_modal.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/components/_modal.scss]: this variable is only used here

**Comment by lory87** [comment]
@FinalAngel yes but because of changes for maxheight and horizontal scrollbar I didn't realized to update the width on modal window. On the current release its ok.

File: djangocms_admin_style/sass/components/_modal.scss (hunk 1 above)
Comment (vxsx) [inline djangocms_admin_style/sass/components/_modal.scss]: oh, okay. i don't know how width of the sidebar is defined, but maybe we can base it off that?

**Comment by vxsx** [comment]
:+1:

**Pull request merged**